pkgs/by-name/st/stormchecker/carl.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+  gmp,
+  ginac,
+  eigen,
+  replaceVars,
+  gtest,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "carl-storm";
+  version = "14.30";
+
+  src = fetchFromGitHub {
+    owner = "moves-rwth";
+    repo = "carl-storm";
+    rev = "${version}";
```

```suggestion
    tag = finalAttrs.version;
```
- unnecessary interpolation
- follow-up to the previous note (finalAttrs) 
- fetchers for forges now support `tag`